src/__tests__/errors/NetworkError.test.ts

```diff
@@ -27,12 +27,12 @@ describe('NetworkError', () => {
     });
 
     it('check BaseError name when message is empty', () => {
-        const ne = new NetworkError('', httpStatusCode, url, exception);
+        const ne = new NetworkError({ message: '', httpStatusCode, url, exception });

```

Comment:
message: '' er hidden feature, som ikke går an å vite om fra utsiden.
Message optional vil være forståelig fra utsiden.

Reply:
Enig, men det er en micro-optimization, og vi må kjekke rydde alle kodebase etter den. @Noggling syns du at det må prioriteseres nå eller skal vi note den og fullføre når litt mer nødvendige tasker er ferdige?

Reply:
Lagt en task på den: https://dev.azure.com/EquinorASA/DT%20%E2%80%93%20Digital%20Twin/_sprints/taskboard/Team%202/DT%20%E2%80%93%20Digital%20Twin/Shared%20Project%20Iterations/Sprint%205-4?workitem=38153

Reply:
Nå som error håndtering ikke er del av echopedia lengre, men skal være generelt for alle, så må det være fullt forståelig for utenforstående/consuming modules. Det er det som er oppgaven med denne US. Du kan godt ha det som egen task, men dette må fikses før dette kan merges til dev.